### PR TITLE
added weight service

### PR DESCRIPTION
**Pull request by Konstantin-Pavlov**

### CODE REVIEW

File: order-service/build/generated/sources/annotationProcessor/java/main/com/coungard/mapper/WeightMapperImpl.java
Comment (coungard) [inline order-service/build/generated/sources/annotationProcessor/java/main/com/coungard/mapper/WeightMapperImpl.java:1]: This class must be unstagable

File: order-service/src/main/java/com/coungard/config/SecurityConfig.java
Comment (coungard) [inline order-service/src/main/java/com/coungard/config/SecurityConfig.java:1]: This file is already formatted

File: order-service/src/main/java/com/coungard/controller/ControllerAdvice.java
Comment (coungard) [inline order-service/src/main/java/com/coungard/controller/ControllerAdvice.java:1]: RestControllerAdvice is already exists in exception package

File: order-service/src/main/java/com/coungard/controller/OrderController.java
Comment (coungard) [inline order-service/src/main/java/com/coungard/controller/OrderController.java:1]: No need format here

**Review by coungard — CHANGES_REQUESTED** [review]
Need fixes

File: order-service/src/main/java/com/coungard/controller/OrderController.java
Comment (Konstantin-Pavlov) [reply order-service/src/main/java/com/coungard/controller/OrderController.java:1]: done

File: order-service/src/main/java/com/coungard/controller/ControllerAdvice.java
Comment (Konstantin-Pavlov) [reply order-service/src/main/java/com/coungard/controller/ControllerAdvice.java:1]: delete this class

File: order-service/src/main/java/com/coungard/config/SecurityConfig.java
Comment (Konstantin-Pavlov) [reply order-service/src/main/java/com/coungard/config/SecurityConfig.java:1]: fix formatting

File: order-service/build/generated/sources/annotationProcessor/java/main/com/coungard/mapper/WeightMapperImpl.java
Comment (Konstantin-Pavlov) [reply order-service/build/generated/sources/annotationProcessor/java/main/com/coungard/mapper/WeightMapperImpl.java:1]: delete from git index and github

**Pull request merged**